### DISCUSSION TITLE
After auto-cancelling my PayPal sponsorships, they've also delisted me as an active sponsor for projects I've sponsored a year in advance

### DISCUSSION THREAD

**Discussion by hmans** [community/community#49502] [Other Feature Feedback, Questions, & Ideas]
I know that nobody at GitHub is monitoring this forum, I'm just posting this here to vent, and to find out if other people have been affected in the same way.

I knew GitHub was going to auto-cancel my existing sponsorships on February 23 (because PayPal), it's fine. But on January 28, as part of my yearly GitHub bill, they charged me for yearly sponsorships (as intended), meaning I am sponsoring these projects until January 2024.

**But on February 23, they immediately delisted me as an active sponsor of these projects.**

Being visible as a sponsor is not the most important thing in the world, but if this happened to me, it will have happened to other folks, too; and consider that with some projects, certain perks and integrations are tied to a user's sponsorship status.

GitHub, **this is not okay**. Please fix this, or issue prorated refunds so we can restart the sponsorships through another payment method.

PS.: yes, I've already opened a support ticket with them. I don't have high hopes for a useful response, but let's find out.

**Comment by emmaviolet** [comment]
Hi @hmans. Thanks for raising this. Just to say this shouldn't have happened, and we're looking into it. Will get back.

**Reply by hmans** [reply]
Any news?

The projects I've sponsored one year in advance through Paypal are still not listing me as a sponsor. Am I getting a refund?
